mandatory/3-function-output.js

```diff
@@ -11,6 +11,9 @@ function combine2Words(word1, word2) {
 function concatenate(firstWord, secondWord, thirdWord) {
   // Write the body of this function to concatenate three words together.
   // Look at the test case below to understand what this function is expected to return.
+  let firstConcat =firstWord.concat(' '+ secondWord)
```

Comment:
Reviewing as requested to pass coursework. Minor things to suggest. Please keep spacing consistent. firstConcat =firstWord. on this line there is no space after equals sign, but in following line, there is a space.

Reply:
Thank your for your guidance. I changed it